css/styles.css

```diff
@@ -0,0 +1,234 @@
+/* pasek nawigacji------------ */
+nav .logo {
+    font-family: 'Pacifico', cursive;
+    font-size: xx-large;
+    margin-left: 8%;
+flex-basis: 40%;
+}
+nav .menu a{
+    font-family: 'Varela Round', sans-serif;   
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: bold;    
+}
+nav{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;    
+}
+
+/* pierwsza stronka-------------- */
+.okno{
+    width: 100vw;
+    height: 100vh;
+    background-image: url(/photo/splash-milk-strawberry.jpg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin: 0;
+}
+
+.textokno{
+    width: 30%;
+    height: 10%;
+    position: relative;
+    left: 10%;
+    top: 10%;
+
+}
+.getfree button{height:50px;width: 250px; background-color: rgb(255, 115, 0); font-size: x-large; box-shadow: black 2px 2px 5px; border-width: 0px; color: rgb(243, 236, 236);font-weight: 100; font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif; text-transform: uppercase; position: relative; left: 10%; top: 150px;}
+
+/* druga stronka */
+main{
+    width: 100vw;
+    height: 100vh;
+    background-image: url(/photo/maliny.jpg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+margin: 0;}
+
+.maliny {
+font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    width: 30%;
+height: 60%;
+position: relative;
+left: 60%;
+top: 30%;
+color: white;
+font-size: xx-large;
+}
+.aboutus p {font-size: large;}
+
+/* trzecia stronka-------- */
+section {
+    margin: 0;
+    width: 100vw;
+    height: 100vh;
+    background-image: url(/photo/veggies.jpg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+}
+section{display: inline-flex;
+flex-direction: row-reverse;}
+
+.services {margin: 0;
+width: 50vw;
+height: 50vw;
+display: inline-flex;
+align-items: center;
+justify-content: center;
+font-size: xx-large; color:white; font-family:'Segoe UI',Tahoma,Geneva, Verdana, sans-serif;}
+
+/* GUZIKI 3 stronki  */
+
+.guziki {
+    width: 50vw;
+    height: 100vh;
+    display: inline-flex;
+    flex-direction: column;
+    align-items: flex-end;
+    justify-content: center;
+    row-gap: 2%;
+    
+}
+.box1, .box2, .box3, .box4{width: 20vw; height: 10vh;
```

Comment:
Use more self explainatory names for classes than box1, box2... Also if box1, box2 etc. have the same styles we can just have one class like `box` (but change the name because I don't know what it referrs to). And each box would have the same styles